SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1093,6 +1089,7 @@ public void purchase(String item, String currency, int cost, int quantity) {
         }
     }
 
+    @Override
```

Comment:
If the interface method signature changes then this makes it far easier to refactor.